### DISCUSSION TITLE
[Help]: How to handle missing images?

### DISCUSSION THREAD

**Discussion by speryt** [davidjerleke/embla-carousel#864] [Help] [answered]
### Summary

I'm trying to add some graceful handling of missing images. By missing I mean that url for the image is returning 404.
With regular image I could try `on:error={doSomething()}`, but I don't see how this can be used with embla. Any suggestions?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [X] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
@s-peryt thanks for your question. I don’t understand the question because Embla doesn’t interfere with your markup? It lets you choose whatever HTML you want. Have you read the [docs](https://www.embla-carousel.com/get-started/svelte/)?

Just put anything you want, images or anything else inside the slide elements (with the class embla__slide in this example):

```HTML
<script>
  import emblaCarouselSvelte from 'embla-carousel-svelte'
</script>

<div class="embla" use:emblaCarouselSvelte>
  <div class="embla__container">
    <div class="embla__slide">Slide 1</div>
    <div class="embla__slide">Slide 2</div>
    <div class="embla__slide">Slide 3</div>
  </div>
</div>
```

**Comment by speryt** [comment]
Oh, I see my mistake. Sorry for that. Of course - I can just operate on individual images inside embla. I must have short brain freeze reading my code. Thanks for pointing that out.

**Reply by davidjerleke** [reply]
@s-peryt no worries. Enjoy!
